lib/errors/course_update_error_handling.rb

```diff
@@ -20,4 +23,21 @@ def report_course_exception_sentry(error, level, extra, sentry_tag_uuid)
       Raven.capture_exception error, level: level, extra: extra
     end
   end
+
+  # Building optional params for Replica and OresApi which require
+  # response body for Oj::ParseError to be saved in CourseErrorRecord
+  # WikiApi does not have Oj::ParseError as a typical error, which is
+  # why it does not use this method
+  def build_optional_params(error, optional_params)
+    # Does processing only if optional_params[:build] is true, which is
+    # true for Replica and OresApi
+    return optional_params unless optional_params[:build] == true
```

Comment:
You don't need to equate it with true. You can just do `unless optional_params[:build]`